lib/OpenQA/WebAPI/Plugin/ObsRsync/Controller/Folders.pm

```diff
@@ -196,4 +197,26 @@ sub forget_run_last {
     return $self->render(json => {message => "error $!"}, status => 500);
 }
 
+sub latest_test {
+    my $self  = shift;
+    my $alias = $self->param('alias');
+    my $full  = $self->param('full');
+    my $id    = $self->obs_rsync->get_last_test_id($alias);
+    return undef unless $id;
+
+    return $self->render(json => {id => $id}, status => 200) unless $full;
+    my $result = '';
+    my $code   = 500;
+    try {
+        my $job = $self->schema->resultset("Jobs")->single({id => $id});
```

Comment:
We usually don't use a try-catch block for database queries like this. The exception are usually transactions which might go wrong. So what's the reason this might go wrong here (except everything is on fire)?

Besides, you could also write the query as `$self->schema->resultset('Jobs')->find($id)`.

Reply:
Don't you think that 'everything is on fire' is enough reason?